www/docs/en/dev/guide/platforms/android/plugin.md

```diff
@@ -72,7 +72,7 @@ Otherwise, the plugin may compile but still be unavailable to Cordova.
 One instance of a plugin object is created for the life of each
 `WebView`. Plugins are not instantiated until they are first
 referenced by a call from JavaScript, unless `<param>` with an `onload`
-`name` attribute is set to `"true"` in `config.xml`. E.g.:
+`name` attribute is set to `"true"` in `config.xml`. For example,
 
     <feature name="Echo">
         <param name="android-package" value="<full_name_including_namespace>" />
```

Comment:
I believe the `"android-package"` param is used for defining the Java package for your app so that the directory structure gets created correctly. See an example usage here: https://github.com/apache/cordova-plugin-camera/blob/master/plugin.xml#L64